tests/tests_geomstats/test_aac.py

```diff
@@ -0,0 +1,79 @@
+from tests.conftest import Parametrizer, TestCase, np_backend
+from tests.data.aac_data import (
+    AACFrechetMeanTestData,
+    AACGGPCATestData,
+    AACRegressorTestData,
+    MaxIterTestData,
+)
+
+IS_NOT_NP = not np_backend()
+
+
+class _MeanEstimatorTest(TestCase):
+    def test_fit(self, estimator, X, expected):
+        estimate_ = estimator.fit(X).estimate_
+
+        dist = estimator.metric.dist(estimate_, expected)
+        self.assertAllClose(dist, 0.0)
+
+
+class _GeodesicEstimatorTest(TestCase):
```

Comment:
Ditto: `GeodesicRegressionTest` if `GeodesicRegression` is the class that is being tested.

Reply:
Aaah just realized you meant GPCA here (Geodesic estimator made me think about geodesic regression where you estimate a geodesic indeed)

Then GGPCATest of PCATest?